packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -388,6 +414,27 @@ export default function TypographyPanel( {
 					/>
 				</ToolsPanelItem>
 			) }
+			{ hasTextColumnsControl && (
+				<ToolsPanelItem
+					className="single-column"
+					label={ __( 'Text columns' ) }
+					hasValue={ hasTextColumns }
+					onDeselect={ resetTextColumns }
+					isShownByDefault={ defaultControls.textColumns }
+					panelId={ panelId }
+				>
+					<NumberControl
```

Comment:
Since this uses the `NumberControl` component, do we need the separate `TextColumnsEdit` component?

Reply:
No. Thanks 👍